docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,415 @@
+---
+id: build-parachains-rococo
+title: Building Parachains on Rococo
+sidebar_label: Building Parachains on Rococo
+---
+
+[Rococo](https://github.com/paritytech/cumulus#rococo-crown) is a Polkadot testnet built for testing
+parachains. Rococo utilizes Cumulus and HRMP (Horizontal Relay-routed Message Passing) in order to
+send transfers and messages between parachains.
+
+## Setting the Environment
+
+To connect to the Rococo testnet, we need to prepare the environment. This command will install
+Substrate onto your machine.
+
+```zsh
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+Make sure you have the Polkadot repository cloned locally and checkout to the `fd4b176` commit.
+
+```
+git clone https://github.com/paritytech/polkadot.git
+cd polkadot
+git checkout fd4b176
+```
+
+Now it's time to build the Relay Chain node. Note that this command could take some time to run on
+your machine.
+
+```
+cargo build --release
+```
+
+## Building the Parachain Template
+
+Next we will be building the parachain template inside the `polkadot` repo, which will be used to
+launch parachains. First, we will clone the parachain template and checkout to the `2710c42` commit.
+
+```
+git clone https://github.com/substrate-developer-hub/substrate-parachain-template.git
+cd substrate-parachain-template
+git checkout 2710c42
+```
+
+Now we will begin building the parachain template in our terminal. Again, this can take some time to
+run on your computer.
+
+```
+cargo build --release
+```
+
+## Defining the Chain Spec
+
+Once you've cloned the Polkadot repo and the parachain template, the next step is to select and
+download a chain spec. A chain specification is needed for your Relay Chain because it contains
+information such as the genesis block, the chain type, and more. Depending on the number of
```

Comment:
How would I know how many validators I have at this point?  I recommend suggesting a single JSON file OR explaining more in-depth.